### PR TITLE
Move tutorials to here and give them new names

### LINKED ISSUE #2: Move tutorial notebooks from DC2-production to here and impose a consistent naming scheme
At title, we can now move the tutorial notebooks within [DC2-production/Notebooks](https://github.com/LSSTDESC/DC2-production/tree/master/Notebooks). While we are moving the notebooks, we can rename them to have a more consistent naming scheme? 

Anyone has strong opinion on how to name these tutorial notebooks? @wmwv @fjaviersanchez @EiffL @jchiang87 @danielsf @evevkovacs?

### PR DESCRIPTION
**Pull request by yymao**
This PR moves the tutorial notebooks from [LSSTDESC/DC2-production/Notebooks](https://github.com/LSSTDESC/DC2-production/tree/master/Notebooks) to here, under the `tutorials` directory. It also [renames](https://github.com/LSSTDESC/DC2-analysis/commit/27fedd3a678) the notebooks according to the discussion in #2.

@drphilmarshall, is it possible to try if your cron job works with this new directory? 

This PR fixes #2

### CODE REVIEW

**Comment by drphilmarshall** [comment]
Before we change the cron job, we need to modify the `beavis-ci` script, to
clone the new repo. You *should* be able to do that and run the script,
passing in your username and GitHub API key - LMK if you are not able to
run the script.

**Comment by drphilmarshall** [comment]
Great! I just updated the cron job to run in this repo instead of DC2-production, as follows:
```
45  4,10,16,22  *  *  * ( cd ~/desc/DC2-analysis/tutorials && ./beavis-ci.sh )
```
I suggest we merge this PR now so that this job can run (on the master branch) in about an hour's time - and if the cron job fails we can issue the fix.

File: tutorials/README.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
+DC2 Notebooks
```
Comment (drphilmarshall) [inline tutorials/README.rst]: This README looks good to me.

**Comment by yymao** [comment]
Thanks, @drphilmarshall.

Per @drphilmarshall's suggestion I'm merging this now. If the beavis CI doesn't run as expected, we will open new issues to address them.

**Pull request merged**